gateway-service/src/main/java/org/zowe/apiml/gateway/security/service/schema/X509Scheme.java

```diff
@@ -115,5 +151,10 @@ private void setHeader(RequestContext context, X509Certificate clientCert) throw
                 }
             }
         }
+
+        private void setErrorHeader(RequestContext context, String value) {
```

Comment:
could be used static method from JwtCommand?

Reply:
making note to fix this in cleanup story

Reply:
fixed